### PR TITLE
Style updates

### PR DESCRIPTION
**Pull request by calebk1**
Dong some style updates on the Platform Edge documentation.

This PR covers the following documents:

- Introduction to Platform Edge
- Mobile Core overview
- Mobile Core API reference

### CODE REVIEW

File: getting-started/introduction-to-aepedge.md [hunk 1]
```diff
@@ -6,27 +6,29 @@ This documentation section is in the process of being moved - please see [https:
...
 
-For instance, XDM events based on take the place of solution APIs such as trackAction and trackState calls to send analytics events. Customers that are not yet ready to fully adopt an XDM only data workflow may opt to keep solution specific API calls, and then create separate XDM calls for important events.
+For instance, XDM events based on ??? can take the place of solution APIs such as trackAction and trackState calls to send analytics events. Customers that are not yet ready to fully adopt an XDM only data workflow may opt to keep solution specific API calls, and then create separate XDM calls for important events.
```
Comment (carlojavier) [inline getting-started/introduction-to-aepedge.md:11]: "???" here on line 11 seems like a typo, or is it something to be filled in later on?

File: getting-started/introduction-to-aepedge.md [hunk 2]
```diff
@@ -6,27 +6,29 @@ This documentation section is in the process of being moved - please see [https:
...
+The new Experience Platform Edge Network extension requires the usage of the latest Adobe Experience Platform SDKs. On iOS, the latest SDK is a new Swift-based SDK that is incompatible with existing Adobe solution extensions such as Adobe Campaign Standard, Adobe Campaign Classic, and Adobe Mobile Services. 
 
-Customers looking to adopt platform-based solutions such as Customer Journey Management or Customer Journey Analytics should consider adopting the new Experience Platform Edge Network extension to simplify sending XDM events to the platform.
+While updates to these solution extensions are in progress, organizations requiring multiple Adobe solutions should first check to ensure that a Swift-compatible or Edge version of the extension exists. For the latest information on supported extensions see [Current SDK Versions](). Third-party vendor extensions should continue to work as expected.
```
Comment (carlojavier) [inline getting-started/introduction-to-aepedge.md:31]: link missing here, but I'm guessing that's to be filled later on

**Review by carlojavier — COMMENTED** [review]
looking good so far!

File: getting-started/introduction-to-aepedge.md (hunk 2 above)
Comment (ahmyr) [reply getting-started/introduction-to-aepedge.md:31]: The Current SDK Versions is this page here: /resources/upgrading-to-aep/current-sdk-versions

File: SUMMARY.md [hunk 3]
```diff
@@ -23,11 +23,11 @@
     * [Lifecycle API reference](using-mobile-extensions/mobile-core/lifecycle/lifecycle-api-reference.md)
     * [Lifecycle metrics](using-mobile-extensions/mobile-core/lifecycle/lifecycle-metrics.md)
     * [Configuration keys](using-mobile-extensions/mobile-core/lifecycle/configuration-keys.md)
-  * [Signal](using-mobile-extensions/mobile-core/signals/README.md)
-    * [Signal API reference](using-mobile-extensions/mobile-core/signals/signal-api-reference.md)
   * [Rules Engine](using-mobile-extensions/mobile-core/rules-engine/README.md)
     * [Rules Engine technical details](using-mobile-extensions/mobile-core/rules-engine/rules-engine-details.md)
     * [Rules Engine consequence details](using-mobile-extensions/mobile-core/rules-engine/rules-engine-consequence-details.md)
+  * [Signal](using-mobile-extensions/mobile-core/signals/README.md)
+    * [Signal API reference](using-mobile-extensions/mobile-core/signals/signal-api-reference.md)
```
Comment (ahmyr) [inline SUMMARY.md:30]: We'll want to keep these in their original position. They are in the order users would typically configure them.

File: getting-started/introduction-to-aepedge.md [hunk 4]
```diff
@@ -6,27 +6,29 @@ This documentation section is in the process of being moved - please see [https:
...
+The new Experience Platform Edge Network extension requires the usage of the latest Adobe Experience Platform SDKs. On iOS, the latest SDK is a new Swift-based SDK that is incompatible with existing Adobe solution extensions such as Adobe Campaign Standard, Adobe Campaign Classic, and Adobe Mobile Services. 
 
-Customers looking to adopt platform-based solutions such as Customer Journey Management or Customer Journey Analytics should consider adopting the new Experience Platform Edge Network extension to simplify sending XDM events to the platform.
```
Comment (ahmyr) [inline getting-started/introduction-to-aepedge.md:31]: Need to keep this sentence in.

File: using-mobile-extensions/mobile-core/README.md [hunk 5]
```diff
@@ -1,34 +1,34 @@
 # Mobile Core
 
-The Mobile Core represents the core Adobe Experience Platform SDK that is required for every app implementation. The core contains a common set of functionality and frameworks, such as Experience Cloud Identity services, data event hub, Rules Engine, reusable networking, disk access routines, and so on, which are required by all Adobe and third-party extensions. The following sections provide additional information about what is provided in the Mobile Core extension.
+Mobile Core is a mandatory Adobe Experience Platform SDK for app implementation. Mobile Core contains a common set of functionality and frameworks, including Experience Cloud Identity services, data event hub, Rules Engine, reusable networking, and disk access routines. The following sections provide additional information about what is provided in the Mobile Core extension.
```
Comment (ahmyr) [inline using-mobile-extensions/mobile-core/README.md:3]: Mobile Core is part of the SDK that is required for all app implementations. The other parts of the SDK are optional. Maybe say "The Mobile Core extension is mandatory for all Adobe Experience Platform app implementations"?

File: using-mobile-extensions/mobile-core/README.md [hunk 6]
```diff
@@ -1,34 +1,34 @@
...
 
 The SDK provides core, extension-independent APIs to facilitate the event tracking of user screens, actions, and PII data. Events generated by these APIs are published to the SDK event hub and are available for use by extensions. For instance, when the Analytics extension is installed, all user actions and app screens event data is sent to the appropriate Analytics reporting endpoints.
 
-For more information, see [Mobile Core API Reference](mobile-core-api-reference.md).
+For more information, please read the [Mobile Core API reference](mobile-core-api-reference.md).
 
 ## Configuration
 
-The configuration extension provides the events in the Event Hub that provide the configuration details for all installed extensions. For more information, see [Configuration](configuration/).
+The configuration extension provides different APIs to set up your application's configuration settings. For more information, please read the documentation on [configuration](configuration/).
```
Comment (ahmyr) [inline using-mobile-extensions/mobile-core/README.md:13]: Double-check with Jiaben Geng that this is okay. Posted in the doc slack for his verification.

Comment (ahmyr) [reply using-mobile-extensions/mobile-core/README.md:13]: Jiabin says it looks good.

File: getting-started/introduction-to-aepedge.md (hunk 1 above)
Comment (emdobrin) [reply getting-started/introduction-to-aepedge.md:11]: We can probably remove the first sentence altogether as it doesn't make sense in this context - this one:
`For instance, XDM events based on ??? can take the place of solution APIs such as trackAction and trackState calls to send analytics events.`

File: SUMMARY.md (hunk 3 above)
Comment (calebk1) [reply SUMMARY.md:30]: Ahh ok, I thought it was a wonky alphabetical order. I'll go fix that back then.

**Pull request closed without merging**